astore/client/commands/commands.go

```diff
@@ -64,10 +67,27 @@ func NewRoot(base *client.BaseFlags) *Root {
 	}
 
 	rc.store.Register(&kcobra.FlagSet{FlagSet: rc.PersistentFlags()}, "")
+	rc.Command.PersistentFlags().StringVarP(&rc.outputFormat, "meta-format", "f", "text", "Meta-data output format. One of [text, json]")
+	rc.Command.PersistentFlags().StringVarP(&rc.outputFile, "meta-file", "m", "-", "Meta-data output file")
```

Comment:
suggestion: instead of having format and file, just have "meta-file"? if empty, no meta-file is created. If non-empty, a meta-file is created based on the extension of the file? See also suggestion below.

I'm also wondering if we should always have the output to the terminal (well, it's disabled easily with >/dev/null), so the meta file is something orthogonal one can ask for (or not).

Reply:
ok

Reply:
done